frontend/karma.conf.js

```diff
@@ -93,14 +93,32 @@ module.exports = function(config) {
 
     // start these browsers
     // available browser launchers: https://www.npmjs.com/search?q=keywords:karma-launcher
-    browsers: [],
+    browsers: ['ChromeHeadless'],
```

Comment:
If I'm right, this change causes Karma to attempt to launch headless Chrome when we run the tests locally in development, too. That would be inconvenient for me, firstly because I have so far managed to avoid installing Chrome and secondly because I like to debug the tests in a headed browser.

I might eventually submit a PR to undo the change on this line, because line 119 already ensures that headless Chrome is used in CI.

Reply:
Oh, I am sorry for that, that was not my intention. I accidentally kept that line, which I added when I was debugging this. (By the way, `ChromeHeadless` works also using Chromium, but I don't know if that is your concern about installing Chrome.)